Source/Core/Core/IOS/ES/Views.cpp

```diff
@@ -74,7 +89,12 @@ IPCCommandResult ES::GetTicketViews(const IOCtlVRequest& request)
 
   const IOS::ES::TicketReader ticket = FindSignedTicket(TitleID);
 
-  if (ticket.IsValid())
+  if (IsUnsupportedIOS(TitleID))
+  {
+    ERROR_LOG(IOS_ES, "GetViews: Dolphin doesn't emulate IOS title %016" PRIx64, TitleID);
+    return GetDefaultReply(ES_EINVAL);
```

Comment:
If the intent is to make it look like there is no ticket for the IOS title, then this should be removed. When no ticket is found, ES doesn't return any particular error code.

Reply:
It has to return something, right? Or does it simply yield control and whatever garbage was left in the return register ends up with the caller?

Reply:
Yep, as far as I can see it just doesn't touch the ioctlv output buffer and returns IPC_SUCCESS.